docs/index.md

```diff
@@ -3,22 +3,64 @@ title: What is this all about?
 sidebar_label: What is this all about?
 ---
 
-<div align="center">
-<img src="./assets/logo.png" alt="Margarita" title="Margarita" width="300"/>
+<div align="left">
+<img src="./assets/logo.png" alt="Margarita" title="Margarita" width="250"/>
 </div>
 
-# Margarita
+<p><br><strong>Margarita is multi-platform <span title="travel">✈️</span> travel app for iOS, Android and Web</strong>
+built with <strong>React Native, GraphQL and <a href="https://partners.kiwi.com/presenting-tequila-revolution-travel-industry/" target="_blank">Tequila API</a></strong></p>
 
-> Showcasing the power of [Tequila](https://partners.kiwi.com/presenting-tequila-revolution-travel-industry/)
+## Demo
 
-This project is a living example of a client app you could start building with Tequila.
+- **Web version**: https://kiwicom-margarita.netlify.com
+- Project in Expo: https://expo.io/@kiwicom-margarita-public/margarita-mobile
 
-- 🕸️ Web version: https://kiwicom-margarita.netlify.com
-- 🎮 GraphQL Playground https://p2kwd3i3a8.execute-api.eu-central-1.amazonaws.com/staging/graphql
-- 🚀 GraphQL Voyager: https://margarita-graphql-voyager.now.sh/
+**If you like this project**, please give us ⭐ on <a href="https://github.com/kiwicom/margarita" targe="_blank">GitHub</a> — it helps!
 
 <div align="center">
-<img src="./assets/screens/search.png" alt="Search" title="Search" width="250"/>
-<img src="./assets/screens/results.png" alt="Results" title="Results" width="250"/>
-<img src="./assets/screens/mmb.png" alt="Bookings" title="Bookings" width="250"/>
+  <img src="./assets/screens/search.png" alt="Search" title="Search" width="250"/>
+  <img src="./assets/screens/results.png" alt="Results" title="Results" width="250"/>
+  <img src="./assets/screens/mmb.png" alt="Bookings" title="Bookings" width="250"/>
 </div>
+
+<img src="./assets/mac.png" alt="Search" title="Search" />
+
+## Devstack
+
+This project follows react-native-web philosophy of **Write Once, Render Anywhere**.
```

Comment:
follows the react-native-web philosophy...

Reply:
Done